src/transfers.rs

```diff
@@ -30,45 +36,134 @@ struct Rule {
     to_stop_id: String,
     transfer_time: Option<u32>,
 }
+/// Represents the type of transfers to generate
+#[derive(PartialEq, Debug)]
+pub enum TransfersMode {
+    /// `IntraContributor` will generate transfers between stop points belonging to the
+    /// same contributor
+    IntraContributor,
```

Comment:
shouldn't it be `All` for normal transferr generation?

Reply:
for now there is no case where we want both generations, do you confirm @prhod

Reply:
yep, and I can't imagine it useful to only add inter and not intra

Reply:
we might want to change the max distance between the two generations

Reply:
So now, this value is never used?